test/e2e/support/reports.js

```diff
@@ -1,15 +1,15 @@
-const reporter = require('cucumberjs-allure-reporter');
-const fs = require('fs');
-const mkdirp = require('mkdirp');
+// const reporter = require('cucumberjs-allure-reporter');
```

Comment:
Probably better to remove these commented-out lines if no longer required.

Reply:
i have removed the contents of the entire file